test/db/esil/arm_16

```diff
@@ -334,19 +334,51 @@ EXPECT=<<EOF
 EOF
 RUN
 
-NAME=lsrs r4, r4, 2
+NAME=lsls r1, r2
```

Comment:
why remove the lsrs test?

Reply:
In fact, the test is still present below starting at line 361
https://github.com/radareorg/radare2/blob/9957224a4dd5ef767b28df48790b3d698112eddc/test/db/esil/arm_16#L361-L383

The check for LSRS was adapted to test the update of the carry flag